cfgov/cfgov/urls.py

```diff
@@ -41,6 +38,16 @@
 from v1.views.documents import DocumentServeView
 
 
+try:
+    from flags.path import flagged_re_path
```

Comment:
This import should be:

```suggestion
    from flags.urls import flagged_re_path
```

Reply:
I'm glad you found this issue. Fixed `django-flags` import path